com.unity.formats.fbx/Documentation~/integration-3ds-max-roundtrip.md

```diff
@@ -0,0 +1,100 @@
+
+### Limitation
+
+You cannot export animation only from Autodesk® 3ds Max®.
+
+### System units
+
+If the system units are not set to centimeters, Autodesk® 3ds Max® prompts you to change them:
+
+![](images/FBXExporter_MaxUnitWarningMsg.png)
+
+* Click __Yes__ to change the system units to centimeters (recommended). This ensures to maintain the scaling on export.
+
+  OR
+
+* Click __No__ to use the current system units (not recommended). The prompt does not appear again for the remainder of the Autodesk® 3ds Max® session or, in the case of a `.max` file, does not appear again for this file.
+
+
+## Re-export the objects to Unity
+
+There are two options available for export in Autodesk® 3ds Max®:
+
+| Option | Description |
+|:-------|:------------|
+| **Export** | Exports both the Models and animation contained in the export sets selected for export. |
+| **Export Model Only** | Exports all Models in the selected export sets, but does not export any animation. |
+
+Both options automatically export the files with the settings and Models you configured during import. To use them:
+
+* From the menu, select **File > Export** and then choose **Export to Unity** or **Export to Unity (Model Only)**.
+
+  ![](images/FBXExporter_MaxUnityMenuExport.png)
+
+To export objects from the desired export set, you can select one or more objects in the set, the set itself, or the corresponding dummy object. In either case, the FBX Exporter exports the entire contents of the set.
+
+If you select multiple dummy objects corresponding to sets or objects from multiple sets, then the FBX Exporter exports each set to its respective file defined in the custom attributes of the set’s dummy object.
+
+In each case, the __Export__ option automatically exports the current Model back to Unity. When you switch back to Unity, your Scene has already been updated.
+
+### Sets and objects to export
+
+Unity export uses the selection sets created on import to determine which objects to export. If you add a new object to the Model, you must also add this new object to the Model’s *UnityExportSet* set.
+
+![UnityExportSets in Autodesk® 3ds Max®](images/FBXExporter_MaxMultipleUnityExportSets.png)
+
+* To edit a *UnityExportSet* set, select **Manage Selection Sets**.
+
+  ![Manage Selection Sets in Autodesk® 3ds Max®](images/FBXExporter_ManageSelectionSets.png)
+
+* To add an object to a set, select the set, select an object and select **Add Selected Objects**.
+
+* To remove an object from a set, select the object in the set and select **Subtract Selected Objects**.
+
+> **TIP:** You can also right-click the UnityExportSets and add or remove objects through the context menu.
+
+![Named Selection Sets in Autodesk® 3ds Max®](images/FBXExporter_MaxNamedSelectionSets.png)
+
+In Autodesk® 3ds Max®, use the **Add Selected Objects** button (red outline) to add objects to the *Wolf_UnityExportSet* set.
```

Comment:
This part is also in the manage-objects-to-export page.
I think some doc was forgotten here after being "moved" to their own page.